app/src/main/java/org/asteroidos/sync/fragments/PositionPickerFragment.java

```diff
@@ -58,6 +59,9 @@ public void onClick(View v) {
                 editor.putFloat(WeatherService.PREFS_ZOOM, zoom);
                 editor.apply();
 
+                // Update the Weather after changing it
+                getActivity().sendBroadcast(new Intent(WeatherService.WEATHER_SYNC_INTENT));
```

Comment:
Somehow, this was supposed to be implemented by https://github.com/AsteroidOS/AsteroidOSSync/blob/master/app/src/main/java/org/asteroidos/sync/ble/WeatherService.java#L63
But it might be not-working for strange reasons.
I have no particular preferences between the WEATEHR_SYNC_INTENT and the OnSharedPreferenceChangeListener but if you choose to use one, please remove the other.

Reply:
Didn't see that. I removed it as it doesn't appear to work whereas the Intent does.